docs/README.md

```diff
@@ -0,0 +1,325 @@
+
+We have made an effort to provide users of EVE with a system that is both
+practically secure and can be deployed in a zero-touch fashion. In order to achieve
+our goals we put forward a [few guiding principles](SECURITY.md).
+
+## EVE Controller
+
+Each Edge Node with EVE running on it can participate in sophisticated
+[Edge Computing scenarious](https://www.zdnet.com/article/10-scenarios-where-edge-computing-can-bring-new-value/)
+ranging from simple [data collection](https://blog.equinix.com/blog/2017/11/29/how-iot-data-collection-and-aggregation-with-local-event-processing-work/)
+all the way to [Edge Analytics and AI](https://www.datanami.com/2019/02/04/exploring-artificial-intelligence-at-the-edge/).
+While EVE provides an incredible amount of functionality to support these use
+cases, it needs to be told exactly how to orchestrate these functional building blocks
+by a *Controller*. In other words, without Controller telling a fleet of Edge
+Nodes what to do, EVE by itself may be capable, but it practically useless.
+
+This is a fundamental point of EVE's design: an end user (lets call her Diane
+The DevOps) trying to implement an Edge Computing scenario never communicates
+directly with a running EVE instance. There are no user names, passwords or a
+concept of "logging in" when it comes to EVE. All interactions that Diane will
+have are going to be through the EVE Controller. Diane expresses her overall
+intent to a controller using a Controller API and it is then the job of a
+controller to build an execution plan for a fleet of Edge Nodes running EVE.
+When each instance of EVE contacts the controller using EVE API it will receive
+its own portion of the overall execution plan (note that EVE API communications
+are  always initiated by EVE, never by the Controller):
+
+![EVE Controller](pics/EVEController.png)
+
+A good way to think about this is that Controller is to EVE what Kubernetes is
+to Docker Engine.
+
+An API between EVE and its Controller is considered a [public API of Project EVE](../api).
+Refer to [API documentation](../api/API.md) for detailed information on API
+end points and message format.
+
+EVE design makes the trust between an EVE instance and its controller extremely
+important to the overall security of your Edge Nodes. An attack vector of
+impersonating an EVE controller or an EVE instance can be used for extremely
+malicious effects (like turning a [fleet of Edge Nodes into a botnet](https://www.cloudflare.com/learning/ddos/glossary/mirai-botnet/)).
+EVE takes great care of making sure that an EVE instance can alwats trust its
+Controller and the Controller can always trust EVE. All of these techniques
+are documented in great details in the [EVE Security section](#security) of
+this document.
+
+A decision of what controller to chose for maintaining a fleet of Edge Nodes
+running EVE currently boils down to two options:
+
+* Hyperscale, commercial [ZEDEDA Controller](https://zededa.com/demo-now/)
```

Comment:
I love ZedCloud, but this is the EVE docs, part of LFEdge which will, eventually, have more. I don't want to turn them off. Can we change this to:

* Hyperscale, commercial, provided by LFEdge members, currently only [ZEDEDA Controller](https://zededa.com/demo-now) (more to follow)

?

Reply:
great point. fixed.